src/NServiceBus.Persistence.CosmosDB/NServiceBus.Persistence.CosmosDB.csproj

```diff
@@ -5,6 +5,7 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
     <LangVersion>8.0</LangVersion>
+    <RootNamespace>NServiceBus.TransactionalSession</RootNamespace>
```

Comment:
that looks off?

Reply:
The idea was to put everything into the same feature namespace of transactional session